app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render 'shared/header', heading:'SIGNUP'%>
+<div  class="w-full flex flex-col justify-center gap-2  px-5 h-96">
+<%= form_for(resource, as: resource_name, class: "w-96", url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
```

Comment:
🚀 Fantastic job with the user authentication 👏🏾  

There are just a few things you need to touch up with your login and registration pages to match the Behance design:

### Registration Page:

Kindly make sure that the registration pages looks exactly like the snapshot below. Please refer to the [Behance website](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for reference.
In this project you are simulating how it would be like working for an organization in the real world. Often as a developer you will be working with graphic designers, therefore, it is crucial that you master how to parse designs to meet client requirements.

Behance | Your App
------------- | -------------
![image](https://github.com/Enning94/RoR-Budget-App/assets/38088818/62767bc2-0091-4ab7-af5b-4723be56e83c) | ![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/1f24b69e-b377-4fe7-8530-d256ececc076)


- As in the design, please make sure that the input fields are 100% of the mobile view width and the are no right and left borders. The should be no top and bottom margins for each filed.
- The sign up and login button should not be present below the form since the sign up button already exists on the navbar.